test/Vegetation/root_test.jl

```diff
@@ -17,29 +17,22 @@ include(joinpath(pkgdir(ClimaLSM), "parameters", "create_parameters.jl"))
 FT = Float64
 
 @testset "Root model integration tests" begin
-    a_root = FT(13192)
-    a_stem = FT(515.5605)
-    b_root = FT(2.1079)
-    b_stem = FT(0.9631)
-    size_reservoir_leaf_moles = FT(16766.2790)
-    size_reservoir_stem_moles = FT(11000.8837)
-    K_max_root_moles = FT(12.9216)
-    K_max_stem_moles = FT(3.4415)
-    z_leaf = FT(12) # height of leaf
+    a_variables = (root = FT(13192), stem = FT(515.5605), leaf = FT(515.5605))
+    b_variables = (root = FT(2.1079), stem = FT(0.9631), leaf = FT(0.9631))
+    size_reservoir_moles = (stem = FT(11000.8837), leaf = FT(16766.2790))
+    K_max_moles = (root = FT(12.9216), stem = FT(3.4415), leaf = FT(3.4415))
     z_root_depths = [FT(-1.0)] # m, rooting depth
-    z_bottom_stem = FT(0.0)
+    delta_z = FT(4.0) # height of compartments
```

Comment:
maybe use a smaller dz? 4 meters is a lot! especially with 11 compartments...

alternatively, @gagnelandmanna can make sure the parameters are physical when she merges her PR...

Reply:
because we may need to adjust the sizes to be smaller if dz is smaller.

Reply:
Yes 4 meters a bit large, unless we have two or three compartments only, maybe 1m is better.

Reply:
Will make sure the params are physical in next PR, but still think its good to make them as realistic as possible to start with!